src/components/UI/Accordion/AccordionItem.tsx

```diff
@@ -11,6 +11,9 @@ interface AccordionItemProps {
   heading: string;
   children: React.ReactNode;
   divider?: boolean;
+  isOpen: boolean;
```

Comment:
changed to integer liao yeah? could change to `openIndex` then?

Reply:
no arh boolean is correct